awx/main/models/inventory.py

```diff
@@ -359,6 +359,8 @@ def get_script_data(self, hostvars=False, towervars=False, show_all=False, slice
             for host in hosts:
                 data['_meta']['hostvars'][host.name] = host.variables_dict
                 if towervars:
+                    if self.kind == 'constructed':
+                        data['_meta']['hostvars'][host.name]['remote_constructed_original_id'] = int(host.instance_id)
```

Comment:
This key doesn't seem to be used anywhere right now that I can see. Are there other places we need to update to make use of this?

I see these right now:

```
[rick@fedora awx]$ rg '\bremote.+id\b'
awx/settings/defaults.py
721:CONTROLLER_INSTANCE_ID_VAR = 'remote_tower_id'
759:CONSTRUCTED_INSTANCE_ID_VAR = 'remote_tower_id'

awx/main/tasks/jobs.py
324:            self.runner_callback.host_map[hostname] = hv.get('remote_tower_id', '')

awx/main/tests/functional/models/test_inventory.py
22:            'remote_tower_id': host.id,
24:            'remote_host_id': host.id,

awx/main/management/commands/inventory_import.py
674:                    for var in ('remote_{}_enabled', 'remote_{}_id'):

awx/main/models/inventory.py
365:                            f'remote_{prefix}_id': host.id,
```